### PR TITLE
Update "The Team" link to pull from astropy repo

### PR DESCRIPTION
**Pull request by eteq**
This PR implements the approach discussed in #19 - the "The Team" page is now pulled from the astropy repo so we don't have to keep it up to date here separately.  You can see the result at http://eteq.github.com/astropy-website/ (or http://eteq.github.com/astropy-website/team.html for the author list page)

### CODE REVIEW

**Comment by eteq** [comment]
@astrofrog @kbarbary

File: replaceteam.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            newcreditslines[0] = ''
+            newcreditslines[1] = 'The Astropy Team'
+            newcreditslines[2] = '================'
+
+            #add the orphan marker
+            newcreditslines.insert(0, ':orphan:')
+
+            # now actually use the new source - this is what sphinx processes
+            source[0] = '\n'.join(newcreditslines)
+            print source
```
Comment (astrofrog) [inline replaceteam.py]: This `print` statement should be removed.

**Comment by astrofrog** [comment]
Looks good, and works for me! It'd be nice to add back links to web pages for people, but that's something that should be done in the core repository (I'll submit a PR for that).

File: replaceteam.py (hunk 1 above)
Comment (eteq) [inline replaceteam.py]: whoops, yeah, that was debug code

**Comment by eteq** [comment]
Alright, I'll merge this now then, but wait until the 0.2 release to actually push it up to the web site (in parallel with #21)

**Pull request merged**